src/libsync/abstractnetworkjob.h

```diff
@@ -41,7 +41,7 @@ class OWNCLOUDSYNC_EXPORT AbstractNetworkJob : public QObject
     Q_OBJECT
 public:
     explicit AbstractNetworkJob(AccountPtr account, const QString &path, QObject *parent = nullptr);
-    virtual ~AbstractNetworkJob();
```

Comment:
@mgallien Why do you want to remove `virtual` from every base classes' destructor? This would lead to an `undefined behaviour` when deleting a derived class' instance through a Base class' pointer, such as base class' destructor will not get called.

Reply:
see my other answer